photon-client/src/components/dashboard/tabs/CropTab.vue

```diff
@@ -0,0 +1,61 @@
+<script setup lang="ts">
+import { useCameraSettingsStore } from "@/stores/settings/CameraSettingsStore";
+import { type AprilTagPipelineSettings, PipelineType } from "@/types/PipelineTypes";
+import PvSlider from "@/components/common/pv-slider.vue";
+import { computed, getCurrentInstance } from "vue";
+import { useStateStore } from "@/stores/StateStore";
+const currentPipelineSettings = computed<AprilTagPipelineSettings>(
```

Comment:
We support crop on more than just AprilTagPipelineSettings right? it's all kinds of pipeline settings? Refactor this to reflect that.